apps/siwe-delegate-manager/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import App from './App'
+
+test('renders learn react link', () => {
```

Comment:
we can create a different ticket to add unit tests to this Safe App in the future. What is your opinion about this?